yt_dlp/extractor/lbry.py

```diff
@@ -317,18 +325,16 @@ class LBRYChannelIE(LBRYBaseIE):
     }]
     _PAGE_SIZE = 50
 
-    def _fetch_page(self, claim_id, url, params, page):
+    def _fetch_page(self, claim_ids, url, params, page):
         page += 1
         page_params = {
-            'channel_ids': [claim_id],
-            'claim_type': 'stream',
             'no_totals': True,
             'page': page,
             'page_size': self._PAGE_SIZE,
         }
         page_params.update(params)
         result = self._call_api_proxy(
-            'claim_search', claim_id, page_params, 'page %d' % page)
+            'claim_search', claim_ids, page_params, 'page %d' % page)
```

Comment:
Since you touched the line, you could adapt it to f-strings:
```suggestion
            'claim_search', claim_ids, page_params, f'page {page}')
```